release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2666,20 +2666,40 @@ This section will continue to be updated over time to provide notes on enhanceme
 
-Issued: 2024-2-27
+Issued: 2024-3-13
 
-{product-title} release {product-version}.0, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7198[RHSA-2023:7198] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7201[RHSA-2023:7201] advisory.
+{product-title} release {product-version}.0, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:1210[RHSA-2024:1210] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:1213[RHBA-2024:1213] advisory.
 
 Space precluded documenting all of the container images for this release in the advisory.
 
 You can view the container images in this release by running the following command:
 
 [source,terminal]
 ----
-$ oc adm release info 4.15.0 --pullspecs
+$ oc adm release info 4.15.2 --pullspecs
 ----
-//replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
+
+[id="ocp-4-15-2-known-issues"]
+==== Known issues
+
+* Providing a performance profile as an extra manifest at Day 0 did not work in 4.15.0 but is now possible in 4.15.2 with the following limitation.
++
+The installation of OCP might fail when a performance profile is present in the extra manifests folder and targets the primary or worker pools. This is caused by the internal install ordering that processes the performance profile before the default primary and worker `MachineConfigPools` are created. It is possible to workaround this issue by including a copy of the stock primary or worker `MachineConfigPools` in the extra manifests folder. (link:https://issues.redhat.com/browse/OCPBUGS-27948[*OCPBUGS-27948*], link:https://issues.redhat.com/browse/OCPBUGS-29752[*OCPBUGS-29752*])
+
+[id="ocp-4-15-2-bug-fixes"]
+==== Bug fixes
+
+* Previously, when updating to {product-title} 4.15 the CatalogSource never refreshed which caused the optional operators to fail to update. With this release, the image pull policy is changed to `Always` which enables the optional operators to update correctly. (link:https://issues.redhat.com/browse/OCPBUGS-30193[*OCPBUGS-30193*])
+
+* Previously, the `nodeStatusReportFrequency` was linked to the `nodeStatusUpdateFrequency`. With this release, the `nodeStatusReportFrequency` is set to 5 minutes. (link:https://issues.redhat.com/browse/OCPBUGS-29797[*OCPBUGS-29797*])
+
+* Previously, under certain conditions the Installer would fail with the error message: `unexpected end of JSON input`. With this release, the error message is clarified and suggests users set the `serviceAccount` field in the install-config.yaml to fix the issue. (link:https://issues.redhat.com/browse/OCPBUGS-29495[*OCPBUGS-29495*])
```

Comment:
```suggestion
* Previously, under certain conditions, the installer would fail with the error message `unexpected end of JSON input`. With this release, the error message is clarified and suggests users set the `serviceAccount` field in the `install-config.yaml` file to fix the issue. (link:https://issues.redhat.com/browse/OCPBUGS-29495[*OCPBUGS-29495*])
```